sklearn/utils/_estimator_html_repr.css

```diff
@@ -0,0 +1,404 @@
+#$id {
+  /* Definition of color scheme common for light and dark mode */
+  --sklearn-color-text: black;
+  --sklearn-color-line: gray;
+  /* Definition of color scheme for unfitted estimators */
+  --sklearn-color-unfitted-level-0: #fff5e6;
+  --sklearn-color-unfitted-level-1: #f6e4d2;
+  --sklearn-color-unfitted-level-2: #ffe0b3;
+  --sklearn-color-unfitted-level-3: chocolate;
+  /* Definition of color scheme for fitted estimators */
+  --sklearn-color-fitted-level-0: #f0f8ff;
+  --sklearn-color-fitted-level-1: #d4ebff;
+  --sklearn-color-fitted-level-2: #b3dbfd;
+  --sklearn-color-fitted-level-3: cornflowerblue;
+
+  /* Specific color for light theme */
+  --sklearn-color-text-on-default-background: var(--theme-code-foreground, var(--jp-content-font-color1, black));
+  --sklearn-color-background: var(--theme-background, var(--jp-layout-color0, white));
+  --sklearn-color-border-box: var(--theme-code-foreground, var(--jp-content-font-color1, black));
```

Comment:
The tweak of this PR is to use variable specific to vs-code and jupyter.
So, I would considered it a hack since we have no guarantee that it does not change apart from if it changes, it would break all the available themes as well.

Secondly, I don't know we could scale to other IDE out there (PyCharm, etc.).